doc/src/devdocs/external_profilers.md

```diff
@@ -0,0 +1,66 @@
+# External Profiler Support
+
+Julia provides explicit support for some external tracing profilers, enabling you to obtain a high-level overview of the runtime's execution behavior.
+
+The currently supported profilers are:
+- [Tracy](https://github.com/wolfpld/tracy)
+- [ITTAPI](https://github.com/intel/ittapi)
```

Comment:
Probably worth calling this "ITTAPI (VTune)" for those who don't know that they're effectively the same thing